examples/styled-components-as-styled-engine/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "material-ui-with-styled-components",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.html",
+  "scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
```

Comment:
I don't think that we should use Parcel for the example. it's barely used by the community. How about a `create-react-app-next-styled-component` example?

Reply:
For preact we ejected https://github.com/mui-org/material-ui/tree/next/examples/preact. Maybe there is away to not eject?

Reply:
Maybe react-app-rewired can help

Reply:
Let me try the `react-app-rewired`, I think it's the best option if it works 👍

Reply:
Updated, can be done using only `react-app-rewired` and `customize-cra`